packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -348,14 +348,35 @@ export interface IFluidDataStoreContextEvents extends IEvent {
 	(event: "attaching" | "attached", listener: () => void);
 }
 
+//* TODO: Find final home for these
+
+export interface IProvideFluidInternalReferenceInfo {
+	//* Make required here?
+	IFluidInternalReferenceInfo?: IFluidInternalReferenceInfo;
+}
+
+//* TODO: Consider converging this type with UnreferencedStateTracker (e.g. have UnreferencedStateTracker implement this interface)
+/**
+ * Info that may be provided by a Fluid Object regarding whether it's referenced or not by other
+ * objects within this container.
+ */
+export interface IFluidInternalReferenceInfo extends Partial<IProvideFluidInternalReferenceInfo> {
+	/** Server timestamp in Unix Epoch format of the point in time (in the op stream) this was detected as unreferenced */
+	unreferencedTime?: number;
```

Comment:
nit: `unreferencedTimestampMs` so looking at this you would know this is a timestamp in milliseconds.